src/store/student-store.js

```diff
@@ -7,7 +6,7 @@ import CheckinStore from './checkin-store';
 import ModelStore from './model-store';
 import { isBeforeNow, dueDateToTimestamp } from '../lib/clock';
 
-const ACTIVE_STATUS = config.get('student.active_status');
+const ACTIVE_STATUS = 'C - Current';
```

Comment:
That wasn't me... git's been commiting weird things on my machine. I'll remove that from this PR